lib/Screens/AuthRegistration/body.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/cupertino.dart';
+
+class Body extends StatefulWidget {
+  @override
+  _BodyState createState() => _BodyState();
+}
+
+class _BodyState extends State<Body> {
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Container(
+          height: double.infinity,
```

Comment:
Probably, here you can use the SizedBox instead https://api.flutter.dev/flutter/widgets/SizedBox-class.html